packages/tea-map-cache/src/index.ts

```diff
@@ -12,30 +13,35 @@ const { getTeaBucketPath } = require('./tea');
  * @param {string} event.s3Key        - Key to write .json tea map cache file to
  * @returns {Promise<Object>}         - A bucketmap object {bucket1: mapping1, bucket2: mapping2}
  */
-async function handler(event) {
+
+export const handler = async (event: {
+  bucketList: string[],
+  s3Bucket: string,
+  s3Key: string
+}): Promise<bucketMapObject> => {
   const { bucketList, s3Bucket, s3Key } = event;
   if (!bucketList || !s3Bucket || !s3Key) {
     throw new Error('A bucketlist and s3 bucket/key must be provided in the event');
   }
-
-  const s3 = new AWS.S3();
+  const teaEndPoint = process.env.TEA_API;
+  if (typeof teaEndPoint !== 'string') {
+    throw new TypeError('process.env.TEA_API must be defined as a string to use this lambda');
+  }
 
   const bucketMapPromises = event.bucketList.map(async (bucket) => ({
-    [bucket]: await getTeaBucketPath({ bucket, teaEndPoint: process.env.TEA_API })
+    [bucket]: await getTeaBucketPath({ bucket, teaEndPoint })
   }));
   const bucketMapObjects = await Promise.all(bucketMapPromises);
 
   const bucketMap = bucketMapObjects.reduce(
     (map, obj) => Object.assign(map, obj), {}
   );
 
-  await s3.putObject({
+  await s3PutObject({
     Bucket: s3Bucket,
     Key: s3Key,
     Body: JSON.stringify(bucketMap)
-  }).promise();
+  });
   console.log(`Wrote bucketmap ${JSON.stringify(bucketMap)} to ${s3Bucket}/${s3Key}`);
```

Comment:
We have the logger package imported so why not use that and get rid of the eslint disable for console statements?

Reply:
I can't think of a good reason not to: 0e51ae993